sse_client/event.go

```diff
@@ -0,0 +1,119 @@
+package sse_client
```

Comment:
You can still use Decoder (https://github.com/donovanhide/eventsource/blob/master/decoder.go) instead of implementing it yourself.